subsquid/src/processor.ts

```diff
@@ -1,27 +1,27 @@
 import { SubstrateProcessor } from "@subsquid/substrate-processor";
 import { TypeormDatabase } from "@subsquid/typeorm-store";
 import { archive, chain } from "./config";
-import {
-  PabloLiquidityAddedEvent,
-  PabloLiquidityRemovedEvent,
-  PabloPoolCreatedEvent,
-  PabloPoolDeletedEvent,
-  PabloSwappedEvent,
-} from "./types/events";
-import {
-  processLiquidityAddedEvent,
-  processLiquidityRemovedEvent,
-  processPoolCreatedEvent,
-  processPoolDeletedEvent,
-  processSwappedEvent,
-} from "./processors/pablo";
-import {
-  processRewardPoolCreatedEvent,
-  processSplitPositionEvent,
-  processStakeAmountExtendedEvent,
-  processStakedEvent,
-  processUnstakedEvent,
-} from "./processors/stakingRewards";
+// import {
+//   PabloLiquidityAddedEvent,
+//   PabloLiquidityRemovedEvent,
+//   PabloPoolCreatedEvent,
+//   PabloPoolDeletedEvent,
+//   PabloSwappedEvent,
+// } from "./types/events";
+// import {
+//   processLiquidityAddedEvent,
+//   processLiquidityRemovedEvent,
+//   processPoolCreatedEvent,
+//   processPoolDeletedEvent,
+//   processSwappedEvent,
+// } from "./processors/pablo";
+// import {
+//   processRewardPoolCreatedEvent,
+//   processSplitPositionEvent,
+//   processStakeAmountExtendedEvent,
+//   processStakedEvent,
+//   processUnstakedEvent,
+// } from "./processors/stakingRewards";
```

Comment:
NIT
Delete commented out code

Reply:
The thing is I need to re-introduce this soon as new pallets are added for the next releases. So wouldn't it be fine to keep it there?

Reply:
I'll keep it for this PR, as now I need to work on stuff for Pablo so some of it will be used, and I'll delete the rest